src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -52,6 +52,8 @@
  * Snowflake via Sink service
  */
 public class SnowflakeSinkTask extends SinkTask {
+  public static final String TASK_INSTANCE_TAG_FORMAT = "SnowflakeSinkTask[ID:{}.{}.{}]";
```

Comment:
Can this go in LoggerHandler?

Reply:
Also lets add a comment here describing how this format looks like in real life. (May be like an example?)

Reply:
Added comment. However the instance tag in the LoggerHandler can be used for things other than the tasks, so I think that task instance tag specific strings should be kept in the task class

Reply:
Additionally the task instance tag should be kept here so we can emit it with telemetry as well